.github/workflows/build_test_push.yaml

```diff
@@ -80,6 +83,22 @@ jobs:
             MININORCAL_OSM_PATH=${{ secrets.MININORCAL_OSM_PATH}}
             MININORCAL_GTFS_PATH=${{ secrets.MININORCAL_GTFS_PATH}}
 
+      # Build the dev server image used in functional tests
+      - name: Build the server
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: false
```

Comment:
can we specify `push: true` here and get rid of the following `Push server-dev image` step?

Reply:
Hmm great question, I vaguely recall hitting some issue with this but I can't find the relevant stack overflow post now lol, so let's try it 🤞

Reply:
Aha, at least the error is explicit 😀 
```
Error: buildx failed with: ERROR: push and load may not be set together at the moment
```